codes/quantum/qubits/Haah Cubic.yml

```diff
@@ -0,0 +1,43 @@
+#######################################################
```

Comment:
Uploading Shor's notes because I don't know how else to do that without putting PDFs into code repo
[shor.pdf](https://github.com/errorcorrectionzoo/eczoo_data/files/7808154/shor.pdf)
.